README.md

```diff
@@ -1,123 +1,81 @@
-
-You can either call `yaml2sbml` via its [command line interface](#Command_line_interface) or within your [python code](#Python): 
-
-### Command line interface
-
-#### yaml2sbml command
-
-To convert an ODE model encoded in a yaml file to SBML using the terminal, go to the `yaml2sbml` folder and run:
+`yaml2sbml`can be `pip`-installed via
 
 ```shell
- yaml2sbml <yaml_input_file> <sbml_output_file>
+pip install yaml2sbml
 ```
+For more infos see the [docs](https://yaml2sbml.readthedocs.io/en/latest/#).
 
-For instance, using the yaml file in the examples folder:
-
-```shell
- yaml2sbml ../examples/ode_input1.yaml ../examples/sbml_out.xml
-```
+## Getting Started
 
-#### yaml2petab command
+* The documentation of the tool is hosted on [read the docs](https://yaml2sbml.readthedocs.io/en/latest/)
+* The [format documentation](https://github.com/yaml2sbml-dev/yaml2sbml/blob/main/doc/format_specification.rst) describes the input YAML. 
 
-If you want to generate PEtab parameter, observable and condition tables, additionally to the SBML file using the terminal, go to the `yaml2sbml` folderand run:
+* Jupyter notebooks containging examples can be found under [`doc/examples`](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples).  Most notably:
+    * [Lotka_Volterra.ipynb](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra/Lotka_Volterra_python/Lotka_Volterra.ipynb) showing the Python package.
+    * [Lotka_Volterra_CLI.ipynb](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra/Lotka_Volterra_CLI/Lotka_Volterra_CLI.ipynb) showing the command line interface.
+    * [Lotka_Volterra_Mock.ipynb](https://github.com/yaml2sbml-dev/yaml2sbml/tree/main/doc/examples/Lotka_Volterra/Lotka_Volterra_Model_Editor/Lotka_Volterra_Model_Editor.ipynb) showing the Model Editor.
 
-```shell
- yaml2petab <yaml_input_file> <petab_output_directory> <model_name>
-```
-
-For instance, again using the yaml file in the examples folder:
-```shell
- yaml2petab ../examples/ode_input1.yaml ../examples/ example_model.xml
-```
+## Basic Syntax
 
 ### Python
 
-Alternatively you can call `yaml2sbml` within your python code via
-
+A YAML model can be translated to SBML/PEtab in Python via
 ```python
 import yaml2sbml
 
-yaml2sbml.yaml2sbml(yaml_file, sbml_file)
-```
+# SBML conversion
+yaml2sbml.yaml2sbml(input_yaml_dir, output_sbml_dir)
 
-Here all inputs ar given as strings.
-
-To generate PEtab files call `yaml2petab` via
+#PEtab conversion
+yaml2sbml.yaml2petab(input_yaml_dir, 
+                     output_petab_dir,
+                     sbml_name)
+```
+### Command Line Interface
 
-```python
-import yaml2sbml
+And in the command line via 
+```shell
+# SBML conversion
+yaml2sbml <yaml_input_file> <sbml_output_file>
 
-yaml2sbml.yaml2petab(yaml_file,
-                     output_dir,
-                     model_name)
+#PEtab conversion
+yaml2petab <yaml_input_file> <petab_output_directory> <model_name>
```

Comment:
Similar to the above comment, the argument names here could also be changed to match the definitions (if this is how the arguments are named in e.g. `yaml2sbml -h` then also fine)

Reply:
They should be :+1: :)